core/qthelper.cpp

```diff
@@ -1653,3 +1653,33 @@ char *copy_qstring(const QString &s)
 {
 	return strdup(qPrintable(s));
 }
+
+// Generate a cylinder-renumber map for use when the n-th cylinder
+// of a dive with count cylinders is removed. It fills an int vector
+// with 0..n, -1, index..count-1. This probably should be moved
```

Comment:
you are using bother `index` and `n` here. That's a cut'n paste error, right?

Reply:
Yes, that's bogus.

Reply:
replaced "index" by "n".